docker-compose.yml

```diff
@@ -783,8 +799,28 @@ services:
         numba: ${NUMBA}
     shm_size: *shm-size
     environment:
-      <<: [ *ccache, *cuda-environment ]
+      <<: [ *ccache, *sccache ] 
+      ARROW_BUILD_UTILITIES: "OFF"
+      ARROW_COMPUTE: "ON"
+      ARROW_CSV: "ON"
+      ARROW_CUDA: "ON"
+      ARROW_DATASET: "ON"
+      ARROW_ENABLE_TIMING_TESTS: "OFF"
+      ARROW_FILESYSTEM: "ON"
+      ARROW_GANDIVA: "OFF"
+      ARROW_GCS: "OFF"
+      ARROW_HDFS: "ON"
+      ARROW_JEMALLOC: "ON"
+      ARROW_JSON: "ON"
+      ARROW_ORC: "OFF"
+      ARROW_PARQUET: "ON"
+      ARROW_PLASMA: "OFF"
+      ARROW_S3: "OFF"
+      ARROW_SUBSTRAIT: "OFF"
+      ARROW_WITH_OPENTELEMETRY: "OFF"
+      SETUPTOOLS_SCM_PRETEND_VERSION:
```

Comment:
Why do we need this?
It seems that other `*-python` don't have this and work without this.

Reply:
Should we add this to other `*-python` too?

Reply:
We don't need this to run in crossbow as the template adds it via `-e `, this was mostly as a convenience for running it locally.